packages/instant/src/util/asset_data_encoder.ts

```diff
@@ -0,0 +1,19 @@
+import { IAssetDataContract } from '@0x/contract-wrappers';
+import { hexSlice, NULL_ADDRESS } from '@0x/utils';
+
+const fakeProvider = { isEIP1193: true } as any;
+
+// instantiate once per app to be more performant
+export const assetDataEncoder = new IAssetDataContract(NULL_ADDRESS, fakeProvider);
+
+/**
+ * Returns the first four bytes of a given hex string.
+ * Does not have any validations, so the hex string MUST
+ * be a valid asset data, or this function returns garbage
+ * without throwing errors.
+ * @param assetData A hex string where the first four bytes are a valid Asset Proxy Id
+ *
+ */
+export function decodeAssetProxyId(assetData: string): string {
+    return hexSlice(assetData, 0, 4);
+}
```

Comment:
Shouldn't something like this be in `order-utils`

Reply:
Maybe but right now nothing else uses it and `instant` doesn't use `order-utils` as a dependency yet...